src/main.rs

```diff
@@ -96,6 +99,8 @@ fn convert(input_file: PathBuf, output_dir: PathBuf, fs: &impl Fs) -> Result<()>
 
                 let markdown = parse_html(item.content());
                 debug!("{}", markdown);
+                let html = transform_html(item.content());
```

Comment:
Looks like lines above are not relevant anymore?

Reply:
Oops, yeah! Fix pushed in 7263b18 . Happy to rebase before merge if you'd like that.